beakerx/beakerx/commands.py

```diff
@@ -0,0 +1,44 @@
+import argparse
+import sys
+import subprocess
+import beakerx
+from .install import install, uninstall
+from .bkr2ipynb import main
+
+def install_subparser(subparser):
+    install_parser = subparser.add_parser('install', help='installs beakerx extension')
+    install_parser.set_defaults(func = install)
+    install_parser.add_argument("--prefix",
+                                help="location of the environment to install into",
+                                default=sys.prefix)
+    return subparser
+
+def uninstall_subparser(subparser):
+    uninstall_parser = subparser.add_parser('uninstall', help='uninstalls beakerx extension')
+    uninstall_parser.set_defaults(func=lambda x : uninstall())
+    return subparser
+
+def bkr2ipynb_subparser(subparser):
+    bkr2ipynb_parser = subparser.add_parser('bkr2ipynb', help='converts beaker notebooks to ipynb format')
+    bkr2ipynb_parser.set_defaults(func=main)
+    bkr2ipynb_parser.add_argument('notebooks', nargs='+',
+                                  help="beaker notebooks to be converted. Enter *.bkr in case you want to convert all notebooks at once.")
+    return subparser
+
+def init_parser():
+    version = 'beakerx %s' % (beakerx.__version__)
```

Comment:
print just the version do not include the name.

Reply:
Additional context, the Python community standardized the output for versions in [PEP 440](https://www.python.org/dev/peps/pep-0440/).

Here is what Django does, and notice they reference PEP 440: https://docs.djangoproject.com/en/2.0/ref/django-admin/#determining-the-version

I see that my example from MarkUpUpDownDown [puts the app name in the version string](https://github.com/MarkUpUpDownDown/markupupdowndown/blob/master/markupupdowndown/commands/__init__.py#L14), so that's my bad for misleading you there. Sorry, @lmitusinski.